warehouse/core/src/test/java/datawave/ingest/mapreduce/job/SplittableRFileInputFormatTest.java

```diff
@@ -0,0 +1,263 @@
+        List<Map.Entry<Key,Value>> data = new ArrayList<>();
+        for (int i = 0; i < 1000000; i++) {
+            addData(data, 1, i);
+        }
+        for (int i = 0; i < 1000; i++) {
+            addData(data, 2, i);
+        }
+        for (int i = 0; i < 1000; i++) {
+            addData(data, 3, i);
+        }
+        tmpFile = createRFile(data);
+        // get the splits from the file
+        List<InputSplit> splits = getSplits(config, new Path(tmpFile.toString()));
+        assertEquals(3, splits.size());
+        verifySplit((RFileSplit) splits.get(0), 0, 127, createKey(1), createKey(1), 1000000);
+        verifySplit((RFileSplit) splits.get(1), 127, 1, createKey(2), createKey(2), 1000);
+        verifySplit((RFileSplit) splits.get(2), 128, 1, createKey(3), createKey(3), 1000);
+    }
+
+    @Test
+    public void testGetSplits_noSizeBasedPreSplit() throws IOException {
+        // set the max split size to be tiny to the FileInputFormat
+        config.setInt("mapreduce.input.fileinputformat.split.maxsize", 10);
+
+        // add more than the max size of data to the file which would otherwise cause the file to be split
+        List<Map.Entry<Key,Value>> data = new ArrayList<>();
+        for (int i = 0; i < 100; i++) {
+            addData(data, 1, i);
+        }
+
+        tmpFile = createRFile(data);
+
+        // link the tmp file to the input format path
+        config.set("mapreduce.input.fileinputformat.inputdir", tmpFile.toString());
+
+        SplittableRFileInputFormat inputFormat = new SplittableRFileInputFormat();
+        JobContext context = new JobContextImpl(config, new JobID());
+        List<InputSplit> splits = inputFormat.getSplits(context);
+
+        // there is only one key used for all data, so it will all be in a single split, unless multiple FileSplit are being generated for the same base file by
+        // FileInputFormat
+        assertEquals(1, splits.size());
+    }
+
+    private Key createKey(int key) {
+        return new Key(String.format("%08x", key));
+    }
+
+    private Value createValue(int value) {
+        return new Value(String.format("%08d", value));
+    }
+
+    private void addData(List<Map.Entry<Key,Value>> data, int key, int value) {
+        data.add(new AbstractMap.SimpleEntry<>(createKey(key), createValue(value)));
+    }
+
+    /**
+     * Verify a split matches expected data. Each value should be 0-index incremented in the split
+     *
+     * @param split
+     *            the split to evaluate
+     * @param startBlk
+     *            the start block of the split
+     * @param blkCount
+     *            the number of blocks in the split
+     * @param first
+     *            the first key of the split
+     * @param last
+     *            the last key of the split
+     * @param keyCount
+     *            the number of keys in the split
+     * @throws IOException
+     * @throws InterruptedException
```

Comment:
Remove the @throws here and the throws statements for `InterruptedException` in the test method declarations above